src/components/MakeWidget/MakeWidget.styles.tsx

```diff
@@ -33,7 +33,7 @@ export const StyledRateField = styled(RateField)`
   justify-content: flex-end;
   margin-left: 1rem;
   margin-bottom: 1rem;
-  width: 50%;
+  width: 100%;
```

Comment:
```suggestion
  width: 50%;
```

This doesn't fix the issue. I made a separate ticket for this, so you can leave this out.

https://github.com/airswap/airswap-web/issues/688